chainerio/filesystems/hdfs.py

```diff
@@ -222,7 +230,17 @@ def _recursive_list(self, full_uri_prefix_offset, path):
 
     def stat(self, path):
         self._create_connection()
-        return self.connection.stat(path)
+        info = self.connection.info(path)
+        stat = HdfsFileStat()
+        stat.filename = info['path']
+        stat.kind = info['kind']
+        stat.mode = info['permissions']
+        stat.size = info['size']
+        stat.owner = info['owner']
```

Comment:
Can you leave some comments to mention that the `stat.owner` and `stat.group` can be string in hdfs case?
https://github.com/chainer/chainerio/blob/faf6d8bb5ad10526436ede13375646f842e7890c/chainerio/io.py#L134
can be a good place

Reply:
Thanks for point it out! I have written docstrings of `FileStats`s to clarify these things.